packages/eslint-plugin/src/rules/no-unsafe-call.ts

```diff
@@ -64,9 +64,15 @@ export default createRule<[], MessageIds>({
             messageId = 'unsafeCallThis';
           }
         }
+
+        const isErrorType = tsutils.isIntrinsicErrorType(type);
+
         context.report({
           node: reportingNode,
           messageId: messageId,
+          data: {
+            type: isErrorType ? '`error`' : '`any`',
```

Comment:
not sure whether "`error`" is the best term in this case
a user might think it refers to the `Error` type in JS, no?

maybe it could be something like "disallow calling unresolved types"?
(probably not the best wording, but something along these lines)

WDYT?

Reply:
Good question. I'd say since #8304 and other PRs are landing in this area with the _"error typed"_ phrasing, let's treat that as a separate followup.